There has been a change in the method safeClose, in the section of code starting on line nr 134.
  
The change is in a catch statement, in a try statement, in the method ```safeClose```, in the class ```Lucene```.
  
The following changes have been made:  
Changed type reference from ```java.io.IOException``` to ```Throwable``` on line 134.  
